cosmo_tester/test_suites/ha/ha_cluster_scenarios_test.py

```diff
@@ -65,16 +65,16 @@ def cluster(
 
         cfy.cluster.nodes.list()
 
-        yield cluster
+        yield hosts
 
     finally:
-        cluster.destroy()
+        hosts.destroy()
 
 
 @pytest.fixture(scope='function')
 def ha_hello_worlds(cfy, cluster, attributes, ssh_key, tmpdir, logger):
     # Pick a manager to operate on, and trust the cluster to work with us
-    manager = cluster.managers[0]
+    manager = cluster.instances[0]
```

Comment:
Ditto. If it's "hosts" everywhere, why is it "cluster" here?

Reply:
Well, here it actually contains a cluster, so I left it... happy to change to `hosts` for consistency though